yarn.lock

```diff
@@ -3443,7 +3443,7 @@ jest-changed-files@^23.4.2:
   dependencies:
     throat "^4.0.0"
 
-jest-cli@^23.6.0:
+jest-cli@^23.3.0:
   version "23.6.0"
```

Comment:
the downgrade has to be more manual due to semver ranges, jest is still the same version